web/blueprints/finance/__init__.py

```diff
@@ -137,6 +138,21 @@ def actions(activity_id):
         } for activity in activity_q.all()])
 
 
+@bp.route('/bank-accounts/import/errors/json')
+def bank_accounts_errors_json():
+    return jsonify(items=[
+        {
+            'name': error.bank_account.name,
+            'fix': {
+                'href': url_for('.fix_import_error',
+                                error_id=error.id),
+                'title': 'korrigieren',
+                'btn_class': 'btn-primary'
+            },
+            'imported_at': '{}'.format(
+                map_or_default(error.imported_at, datetime.date, 'nie'))
```

Comment:
FYI, with the upcoming  [Py3.8 Assignment expressions](https://www.python.org/dev/peps/pep-0572/), the helper function could be inlined to
```python
'imported_at': str(datetime.date(i)) if (i := error.imported_at) is not None else 'nie'
```